packages/libs/react-ui/src/components/Tabsss/Tab.tsx

```diff
@@ -0,0 +1,30 @@
+import classNames from 'classnames';
+import type { FC, ReactNode } from 'react';
+import React from 'react';
+import { selectedClass, tabClass } from './Tabs.css';
+
+export interface ITabProps {
+  children: ReactNode;
+  selected?: boolean;
+  handleClick?: (tabId: string) => void;
+  id: string;
+}
+
+export const Tab: FC<ITabProps> = ({
+  children,
+  selected = false,
+  handleClick,
+  id,
+}) => {
+  if (handleClick === undefined || id === undefined) return null;
+  return (
+    <button
+      className={classNames(tabClass, { [selectedClass]: selected })}
```

Comment:
I don't think you need to add the selected class like this since you can style based on the data-selected attribute

Reply:
I see now that this is the old component